javascript/webapp/src/components/NavPanel/NavPanel.tsx

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+import { Panel } from '@fluentui/react'
+import { memo } from 'react'
+
+export interface NavPanelProps {
+	isOpen: boolean
+	onDismiss: () => void
+}
+
+export const NavPanel: React.FC<NavPanelProps> = memo(function NavPanel({
+	isOpen,
+	onDismiss,
+}: NavPanelProps) {
+	return (
+		<Panel
+			isLightDismiss
+			isOpen={isOpen}
+			onDismiss={onDismiss}
+			closeButtonAriaLabel="Close"
+			headerText="Menu"
+		>
+			<h3>Settings</h3>
+			<h3>Help</h3>
```

Comment:
Can you add a third section with `Link` components to Home, Performance, Debug pages?

Reply:
Will be done in another PR in a different branch